### PR TITLE
Update return types + formatting + typos

### PR DESCRIPTION
**Pull request by thomaschaplin**
In this pull request I have completed the below tasks:

* Added return types to all functions in `mod.ts`
* Applied formatting to `mod.ts` by running `deno fmt`
* Resolved a typo in `mod.ts`
* Applied formatting to `readme.md`
* Resolved a typo in `readme.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ Appwrite playground for Deno is a simple way to explore the Deno integration wit
 
 This playground doesn't include any deno best practices but rather intended to show some of the most simple examples and use cases of using the Appwrite API in your deno application.
 
-## System Requirements 
+## System Requirements
 * A Linux/Windows system deno installed and path is properly set for deno command execution.
-* You have readily available AppWrite running instance (localost in most cases).
+* You have readily available AppWrite running instance (locahost in most cases).
```
Comment (eldadfux) [inline README.md]: => localhost

Comment (thomaschaplin) [reply README.md]: Good spot, fixed in a new commit!

File: spell.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Appwrite
```
Comment (eldadfux) [inline spell.txt]: What does this file use for?

Comment (thomaschaplin) [reply spell.txt]: So sorry, wasn't mean to commit that 😢  Removed it

**Review by W4nn4Die — APPROVED** [review]

**Review by eldadfux — APPROVED** [review]

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like me to send you some cool Appwrite stickers and magnets! Please note that shipping might take longer times because of COVID19.

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

**Pull request merged**